ush/generate_FV3LAM_wflow.sh

```diff
@@ -568,7 +568,8 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-if [ "${RUN_TASK_RUN_FCST}" = "TRUE" ]; then
+if [ "${RUN_TASK_MAKE_OROG}" = "TRUE" ] || \
+   [ "${RUN_TASK_RUN_FCST}" = "TRUE" ]; then
 #
```

Comment:
Why is this change needed?

Reply:
Because when running `make_orog` but not `make_fcst`, it turns out the `FIXam` directory is still needed.  I think we had never tested the App in that configuration before.  And yes, something like this is probably also needed in the authoritative repo's `develop` branch, but again I've lost track and don't know if it's already in there or in one of the PRs that are slated to be merged.

Reply:
Ahh, interesting.  Yep, that was going to be my next question (how is this working in the authoritative repo?).  I would assume the App would be broken without the FIXam directory for make_orog, since that's the case here, so I imagine it's been modified.